src/components/sidebar/connection/ConnectionListItem.vue

```diff
@@ -101,6 +110,14 @@ export default {
     },
     duplicate() {
       this.$emit('duplicate', this.config)
+    },
+    copyUrl() {
```

Comment:
If you make this an `async` function you can use `await this.$copyText` then just try/catch as usual

Reply:
Good to know, thanks!
I changed that.